modules/course_management/course_activator.py

```diff
@@ -0,0 +1,47 @@
+from typing import Literal, Union
+import discord
+from discord_slash.context import SlashContext
+import config
+from modules.course_management.util import sort_single_course
+from modules.error.friendly_error import FriendlyError
+from utils.utils import get_discord_obj, get_id
+
+
+async def activate_course(ctx: SlashContext, channel: discord.TextChannel):
+	"""Move a course from the inactive courses category to the active one."""
+	await __move_course(ctx, channel, "activate")
+
+
+async def deactivate_course(ctx: SlashContext, channel: discord.TextChannel):
+	"""Move a course from the active courses category to the inactive one."""
+	await __move_course(ctx, channel, "deactivate")
+
+
+async def deactivate_all_courses(ctx: SlashContext):
+	"""Move all active courses from the active courses to the inactive one."""
+	category: discord.CategoryChannel = get_discord_obj(
+		config.guild().categories, "ACTIVE_COURSES_CATEGORY"
```

Comment:
strings "ACTIVE_COURSES_CATEGORY" and "INACTIVE_COURSES_CATEGORY" can be extracted to a literal constant to avoid having it be repeated in several places.